testing/citest/tests/kube_smoke_test.py

```diff
@@ -679,11 +699,14 @@ def execute_delete_daemonset_pipeline(self):
       builder = kube.KubeContractBuilder(self.kube_observer)
       (builder.new_clause_builder('Daemonset Deleted', retryable_for_secs=15)
        .get_resources(
-          'daemonsets', extra_args=[self.TEST_APP+'tstv001', '--namespace', self.TEST_NAMESPACE])
-       .contains_path_value('kind', 'DaemonSet'))
+          'daemonsets', extra_args=[self.TEST_APP +
```

Comment:
Here and elsewhere is awkward line break flow.
You seem to break eagerly on syntactic boundaries rather than logically on expression boundaries. Rather than
    <resource>, extra_args=[APP +
                                               POSTFIX, '--namespace',
                                               NAMESPACE]
I'd think this would be more readable:
    <resource>,
    extra_args=[APP + POSTFIX,
                         '--namespace', NAMESPACE]
I follow strict line-break rules, but try to balance compactness and flow.
Both examples use 3 lines, but the second groups parameters together
putting both operands of the '+' on the same line, and associating the key and value of the namespace.

The line break in the + operator for the APP + POSTFIX is a common pattern you use above, as opposed to putting the break before the extra_args to give you more working space in the arg list lines then grouping expressions.

Reply:
Fixed the code and checked in.